README.rst

```diff
@@ -77,6 +77,29 @@ Remember to add a ``SECRET_KEY`` setting!
     python manage.py index
 
 
+Static Files
+------------
+
+Static files are stored in `sitemedia/` and bundled by Webpack. The `webpack-bundle-tracker` plugin generates a JSON manifest file listing the name and location of bundled files. This file — `webpack-stats.json` in production/QA/test and `webpack-stats-dev.json` for local development — is read by Django using `django-webpack-loader` so that the relevant files can be included in the template's `<head>`.
+
+Bundled files and the production JSON manifest are checked into git and will be collected by Django's `collectstatic` command. They should be recompiled and any changes should be committed as part of a release. To recompile bundles for production::
+
```

Comment:
Let's add the `npm install` command somewhere to the development instructions. Maybe after installing python packages?